src/Microsoft.OData.ModelBuilder/Vocabularies/VocabularyTermConfigurationExtensions.tt

```diff
@@ -46,10 +50,26 @@ namespace Microsoft.OData.ModelBuilder
     {
         internal static IEdmExpression ToEdmExpression(this string text)
             => string.IsNullOrEmpty(text) ? null: new EdmStringConstant(text);
```

Comment:
```suggestion
            => string.IsNullOrEmpty(text) ? null : new EdmStringConstant(text);
```
Not part of this PR but please add the spacing so that formatting the generated document does not produce a diff... Realized this after the last feature addition.